src/main/java/com/mojang/brigadier/StringReader.java

```diff
@@ -239,4 +239,21 @@ public void expect(final char c) throws CommandSyntaxException {
         }
         skip();
     }
+
+    public void expect(final String s) throws CommandSyntaxException {
+        if(!canRead(s.length()) || !getRemaining().startsWith(s)) {
+            throw CommandSyntaxException.BUILT_IN_EXCEPTIONS.readerExpectedSymbol().createWithContext(this, s);
+        }
+        this.cursor += s.length();
+    }
+
+    public String expectOption(final String ...opts) throws CommandSyntaxException {
+        for(String s: opts) {
+            try {
+                expect(s);
+                return s;
+            } catch (CommandSyntaxException e) {}
```

Comment:
Style issue: Name the exception `ignored` to make it clear we don't care for it, and make it final.

(I'm sorry - this is my fault for not commiting a style guide yet. I'll do that as soon as I can.)

Reply:
> and make it final.

Now that is a style choice I've never heard of before...?

Reply:
I suppose the JVM handles final things better, so I guess it is a good style choice?